fvm/transactionInvoker.go

```diff
@@ -35,7 +35,7 @@ func (i TransactionInvoker) Process(
 
 	txIDStr := proc.ID.String()
 	var span otelTrace.Span
-	if ctx.Tracer != nil && proc.TraceSpan != nil {
```

Comment:
This boilerplate (this + the next line) is very similar to environment.Tracer.StartSpanFromRoot (the "root" used by the environment is the span that's being created here).  We can probably refactor / reuse parts of StartSpanFromRoot here.

Reply:
good suggestion. I'm keeping `IsSampled()` which I need for my next PR to emit computation effort vector metrics.